code_examples/sdk_examples/src/core_features/getting_started/03_fetch_did.ts

```diff
@@ -1,16 +1,16 @@
 /* eslint-disable prefer-const */
 import * as Kilt from '@kiltprotocol/sdk-js'
+import * as Did from '@kiltprotocol/did'
 
-export async function main(): Promise<Kilt.DidUri | null> {
+export async function main(): Promise<String | null> {
   let apiConfig = Kilt.ConfigService.get('api')
   const encodedKiltnerd123Details =
     await apiConfig.call.did.queryByWeb3Name('kiltnerd123')
 
-  // This function will throw if kiltnerd123 does not exist
   const {
-    document: { uri }
-  } = Kilt.Did.linkedInfoFromChain(encodedKiltnerd123Details)
-  console.log(`My name is kiltnerd123 and this is my DID: "${uri}"`)
+    document: { id }
+  } = Did.linkedInfoFromChain(encodedKiltnerd123Details)
```

Comment:
This isn't an ideal way of doing things, but it's missing from the SDK, so @rflechtner is going to add a new method and then we can update this.

Reply:
Yes, as I explained above, unfortunately (as far as I saw), it’s not included in the new versions of the SDK.